### PR TITLE
Mainnet Readiness

### PR DESCRIPTION
**Pull request by KPrasch**
Based on #1

### CODE REVIEW

**Review by jMyles — APPROVED** [review]
I did not verify contract addresses or dependencies.  I presume that these are already good from the previous version.  If review of this material is desired, let me know and I'll take another look.

I'm pondering whether I actually prefer "Error" to "Oh no" - the latter seems more informative.  ;-)

**Review by vepkenez — CHANGES_REQUESTED** [review]
In my opinion, all of these contract addresses should be in a .env file or at least in a `constants` file.
It should be very easy to audit their correctness without hunting for them throughout the code.

This is the primary concern.

File: src/components/worklock/worklock.js [hunk 1]
```diff
@@ -211,7 +192,7 @@ const WorkLock = ({
                   title="Nu tokens allocation:"
```
Comment (cygnusv) [inline src/components/worklock/worklock.js]:
```suggestion
                  title="NU tokens allocation:"
```

File: src/components/worklock/worklock.js [hunk 2]
```diff
@@ -256,7 +237,7 @@ const WorkLock = ({
             <div className={s.economics_container}>
                <div className={s.economics_top_row}>
                   <div>
-                     <span>Minimal allowed bid</span>
+                     <span>Minimum allowed escrow</span>
                      <h4>{minAllowedBid} ETH</h4>
                   </div>
                   <div>
```
Comment (cygnusv) [inline src/components/worklock/worklock.js:243]: L258 above says "Number of bidders" but I can't comment there. Change to participants.

File: src/services/web3-service.js [hunk 3]
```diff
@@ -286,7 +289,7 @@ export default class ServiceWeb3 {
          tokensAllocated: ethToTokensNu,
          boostingRefund: boostingRefund,
          SLOWING_REFUND: SLOWING_REFUND,
-         аvailableRefund: parseFloat(getAvailableRefund).toFixed(4),
+         availableRefund: parseFloat(getAvailableRefund).toFixed(4),
```
Comment (cygnusv) [inline src/services/web3-service.js:292]: What's the change on this line? Both seem identical, at least from GH.

File: src/services/stake-service.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import Web3 from 'web3';
-import { Token } from '../ethereum/instances/instances';
-import { hexToBytes } from '../utils/utils';
+import {Token} from '../ethereum/instances/instances';
+import {hexToBytes} from '../utils/utils';
 
 const DISPATCHER_ADDRESS = '0xAB51fBDd4Faf6c691884B3A9b475E34E2092aE81';
```
Comment (cygnusv) [inline src/services/stake-service.js]: What's this address?

**Comment by KPrasch** [comment]
@vepkenez 

> In my opinion, all of these contract addresses should be in a .env file or at least in a constants file.
It should be very easy to audit their correctness without hunting for them throughout the code. ...This is the primary concern.

There already is such a module: `instances.js`.  These addresses are not secret, completely public, and will never change. 

To me that means "Hardcoded"

File: src/services/stake-service.js (hunk 4 above)
Comment (vepkenez) [inline src/services/stake-service.js]: @KPrasch what is this one?

**Review by vepkenez — APPROVED** [review]

**Review by cygnusv — APPROVED** [review]

File: src/services/web3-service.js (hunk 3 above)
Comment (KPrasch) [reply src/services/web3-service.js:292]: It fixes an non-ascii "a".

File: src/services/stake-service.js (hunk 4 above)
Comment (KPrasch) [reply src/services/stake-service.js]: Yes - good catch - this is incorrect and from testnet.

Comment (KPrasch) [reply src/services/stake-service.js]: Yes - good catch - this is incorrect and from testnet.

File: src/components/worklock/worklock.js (hunk 1 above)
Comment (KPrasch) [reply src/components/worklock/worklock.js]: :heavy_check_mark:

File: src/components/worklock/worklock.js (hunk 2 above)
Comment (KPrasch) [reply src/components/worklock/worklock.js:243]: :heavy_check_mark:

**Pull request merged**